src/main/java/org/fundacionjala/coding/denis/Ean.java

```diff
@@ -0,0 +1,29 @@
+package org.fundacionjala.coding.denis;
+
+public class Ean {
+    private int[] digitos;
+
+    public boolean kata() {
+        return digitos.length == 13 ? isEan() : false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "false" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1125)